site/gatsby-site/src/components/reports/ReportCard.js

```diff
@@ -143,23 +138,19 @@ const ReportCard = ({
         </div>
         <div className="mt-0 cursor-default select-text">
           <div className="flex">
-            {reportTitle ? (
-              <>{reportTitle}</>
-            ) : (
-              <button
-                className="w-3/4 text-left"
-                onClick={toggleReadMore}
-                onKeyDown={toggleReadMoreKeyDown}
-              >
+            <Link to={`/reports/${item.report_number}`} className="hover:no-underline">
```

Comment:
@clari182 Is this URL correct? When I navigate to https://pr-2756--staging-aiid.netlify.app/cite/658#r3803 and click on the report title, it redirects me to https://pr-2756--staging-aiid.netlify.app/reports/3803/ and it doesn't exist.

<img width="441" alt="image" src="https://github.com/responsible-ai-collaborative/aiid/assets/6564809/c2fd6a9e-61e5-46dd-82f7-5b28a81fe6ac">